airbyte-integrations/connectors/source-rippling/source_rippling/schemas/company_leave_types.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "properties": {
+    "description": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "id": {
+      "type": "string"
+    },
+    "isUnpaid": {
+      "type": "boolean"
+    },
+    "leaveType": {
+      "type": "string"
+    },
+    "name": {
+      "type": "string"
+    }
```

Comment:
Can you null these types? It will help make this connector more robust with api regressions